index.html

```diff
@@ -2,21 +2,22 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Booklist</title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8" />
```

Comment:
- [ ] In the `readme.md`, please include the `Getting started` section with instructions on how to run the project.

Reply:
Thanks for the Review We have added The Required changes to Readme.md file